includes/class-omise-wc-myaccount.php

```diff
@@ -61,31 +61,35 @@ public function register_omise_my_account_scripts() {
 				true
 			);
 
-			wp_enqueue_script(
-				'omise-util',
-				plugins_url( '/assets/javascripts/omise-util.js', dirname( __FILE__ ) ),
-				array( 'omise-js' ),
-				WC_VERSION,
-				true
-			);
-
 			wp_enqueue_script(
 				'omise-myaccount-card-handler',
 				plugins_url( '/assets/javascripts/omise-myaccount-card-handler.js', dirname( __FILE__ ) ),
-				array( 'omise-js' , 'omise-util' ),
+				array( 'omise-js' ),
 				WC_VERSION,
 				true
 			);
 
-			wp_localize_script(
-				'omise-myaccount-card-handler',
-				'omise_params',
-				array(
-					'key'             => Omise()->settings()->public_key(),
-					'ajax_url'        => admin_url( 'admin-ajax.php' ),
-					'ajax_loader_url' => plugins_url( '/assets/images/ajax-loader@2x.gif', dirname( __FILE__ ) )
+			$omise_params = array(
+				'key'                            => Omise()->settings()->public_key(),
+				'ajax_url'                       => admin_url( 'admin-ajax.php' ),
+				'ajax_loader_url'                => plugins_url( '/assets/images/ajax-loader@2x.gif', dirname( __FILE__ ) ),
+				'required_card_name'             => __( 'Card Holder\'s name is a required field', 'omise' ),
+				'required_card_number'           => __( 'Card Number is a required field', 'omise' ),
+				'required_card_expiration_month' => __( 'Card Expiry month is a required field', 'omise' ),
```

Comment:
'Card' probably unnecessary here

Reply:
and the two fields next

Reply:
This error message is mixed with the other invalid form input's error messages.
If possible and not grammatically wrong, as for UX of WC, I think we better explicitly say that this error is coming from the credit card form, not others.

Reply:
Ah, the error message isn't displayed next to the field?